extensions/amp-story/1.0/animation.js

```diff
@@ -325,6 +325,16 @@ export class AnimationRunner {
     });
   }
 
+  /**
+   * Applies the last animation frame.
+   * @return {!Promise<void>}
+   */
+  applyLastFrame() {
+    this.runnerPromise_.then((runner) => {
+      runner.finish();
```

Comment:
I don't think this will work on native animations on Safari with `prefers-reduced-motion`.

Reply:
Edit: @gmajoulet and I tested on iOS Safari and the current implementation works

Reply:
Can you please add a test case so I can confirm? (Not unit test. A document.)